src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -491,7 +491,7 @@ End Class</Code>
             Dim expected = <Code>Class C
     Sub Method()
         Dim a = From q In
-                   {1, 3, 5}
+                                                {1, 3, 5}
```

Comment:
this does not not seem good.  what happened here?

Reply:
ok.  based on your previous comment, this is explained.  that said, this is so weird too.  why did this value end up on the *next* line?  that feels like a bug itself.

Reply:
This was already on its own line, which is a strange situation from the start.